composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "buibr/tmdbapi",
+    "description": "PHP wrapper for The Movie Data Base API version 3",
+    "keywords": ["markdown", "gfm", "markdown-extra", "fast", "extensible"],
+    "type": "library",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Adan GQ",
+            "email": "adan@dont.know",
+            "role": "Developer"
+        },
+        {
+            "name": "Burhan Ibrahimi",
+            "email": "burhan@wflux.pro",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": ">=5.5.0"
```

Comment:
In this repository, it uses the `curl_` extension and it should add the `ext-curl: "*"` inside this block.